.codesandbox/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:lts-alpine
+
+# 1. zsh is default shell on CodeSandbox - without it, the devbox fails to start
+# with a "OCI runtime exec failed" error:
+# ```
+# OCI runtime exec failed: exec failed: unable to start container process: exec: '/bin/zsh': stat /bin/zsh: no such file or directory: unknown
+# ```
+#
+# 2. Git is a dependency of zsh - without it, opening the terminal fails with a
+# "command not found" error:
+# ```
+# "/root/.oh-my-zsh/tools/check_for_upgrade.sh:31: command not found: git"
+# ```
+#
+# TODO: Switch to bash and remove the zsh packages in future if CodeSandbox adds
+# a feature to configure the shell
+# https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294#discussion_r1601759357
+RUN apk update && apk add --no-cache zsh git
+
+# Avoid prompts during package installation using Corepack with PNPM. This prevents the
```

Comment:
pnpm is lowercase, right?

Reply:
Changed pnpm to lowercase
[bb27ce3](https://github.com/upleveled/express-guest-list-api-memory-data-store/pull/294/commits/bb27ce3f71bdc676cf8a95372691ac1c61bee442)